### PR TITLE
Use JSX elements instead of parameterless functions + support for l10n customizations

### PR DESCRIPTION
**Pull request by apeltola**
Modified l10n files so that parameterless functions are replaced with direct JSX elements.
React.isValidElement used for detecting JSX elements in localization files.

### CODE REVIEW

File: frontend/src/lib-customizations/common.tsx [hunk 1]
```diff
@@ -50,4 +52,19 @@ export const absenceColours = {
   PRESENCE: colors.greyscale.white
 }
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export const translationsMergeCustomizer = (
```
Comment (apeltola) [inline frontend/src/lib-customizations/common.tsx:56]: This is the most important change: allows overriding JSX elements in city specific implementations.

Comment (akheron) [reply frontend/src/lib-customizations/common.tsx:56]: You can avoid the `eslint-disable` like this:

```
 export const translationsMergeCustomizer = (
  origValue: Record<string, unknown>,
  customizedValue: unknown
): unknown => {
```

**Review by akheron — CHANGES_REQUESTED** [review]

**Comment by akheron** [comment]
Please merge on top of the latest master to get the build to pass

File: frontend/src/lib-customizations/common.tsx (hunk 1 above)
Comment (apeltola) [reply frontend/src/lib-customizations/common.tsx:56]: Maybe even better to have it like this?

```
export const translationsMergeCustomizer = (
  origValue: Record<string, unknown>,
  customizedValue: Record<string, unknown>
): Record<string, unknown> | undefined => {
```

Comment (akheron) [reply frontend/src/lib-customizations/common.tsx:56]: Yeah, sure 👍

**Review by akheron — APPROVED** [review]

**Pull request merged**